discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -128,23 +128,39 @@ def set_cluster_name(self, cluster_name):
     def select_installation_disk(self, hosts_with_disk_paths):
         self.api_client.select_installation_disk(self.id, hosts_with_disk_paths)
 
-    def set_ocs(self, ocs_enabled):
-        logging.info(f'Enabling Ocs to:{ocs_enabled} for cluster: {self.id}')
-        self.set_operator('ocs', ocs_enabled)
+    def set_ocs(self, properties=None):
+        self.set_olm_operator('ocs', properties=properties)
```

Comment:
why remove the logging entry? too verbose or already covered elsewhere?

Reply:
The log entry is now in methods `set_olm_operator` and `unset_olm_operator`.